benchdata/simpledata.json

```diff
@@ -0,0 +1 @@
"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,82,-82,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,83,-83,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,84,-84,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":false},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,85,-85,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,86,-86,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,87,-87,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,88,-88,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":false},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,89,-89,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,90,-90,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,91,-91,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,92,-92,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":false},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,93,-93,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,94,-94,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,95,-95,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":false,"e0":"enum3","l0":[0,96,-96,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":false},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,97,-97,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,98,-98,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,99,-99,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":true},{"i1":16,"i2":127,"b1":true,"e0":"enum3","l0":[0,100,-100,127],"i0":32,"e1":"enum7","s0":{"f1":3.1415926535898,"f0":3.14},"t0":"hello","i3":65536,"b0":false}]}
```

Comment:
I just used the output from the library itself it was never designed with pretty printing. I did start out with making some fakedata like json.lua https://github.com/rxi/json.lua/blob/master/bench/bench_encode.lua#L15 to bench but I thought it would be more interesting to use the same data as capnproto.

Reply:
I'm not sure what the ramifications of this are, but this file doesn't end with a newline. You can see github warning you about this...

Reply:
All the files actually. Is this a windows thing?

Reply:
It's not a Windows thing -- editors can be configured to add/not add newlines. I don't think it's a big deal.

Reply:
[Re: the one-line formatting. OK, since that's what the tool produced, let's leave it as-is.]

Reply:
ok